app/controllers/donations_controller.rb

```diff
@@ -102,53 +102,47 @@ def destroy
     redirect_to donations_path
   end
 
-  private
```

Comment:
I think it's an overall Ruby habit and not specific to Rails. I used to do it that way too, because it used to be the only way to specify it, and sometimes still do. However, the inline option language addition is a clear improvement, because:

* it shows the reader the private status of the method without the need to scan upwards in the file for a `private` line
* it guarantees that if that method definition, or the `private` line, is moved, the method will not lose its private status
* it does not become more difficult to see its private status as the number of private methods increases

This is evidenced by the fact that I missed that the method was private and had to modify my test to use `send`, requiring additional developer time. Sure, I should have scanned upwards for `private`, but with the inline approach I wouldn't have needed to.

So from my point of view, doing it the way we have always done it is a habit that we would be wise to change. I am willing to revert this change, but just want to push back a little with my reasoning and ask if this is something that the team really wants to do.

Reply:
I think your reasoning is absolutely correct, and I definitely appreciate the pushback. I'm still worried about this breaking rails convention, so I'll ask a few other people's opinions then get back to you on it :)

Reply:
@keithrbennett I understand your reasoning, and I respect the desire to improve developer happiness and ease. 

I also think it's an out-of-scope change for this PR, and creates asymmetry in how we address `private` methods in this class versus how we address them in other classes -- if you'd like to make a separate issue and write a PR to address *all* of the classes (so that it's uniform), that would be fine!

Reply:
@armahillo Got it. I've reverted the 'private' changes. There was some git weirdness, the branch says "This branch is 9 commits ahead, 7 commits behind master. " I'm looking into this. If I can't find a better way, would it be ok to recreate my branch with a single commit, and force push it up to Github?

Reply:
@keithrbennett Looks like you figured it out!